vncdotool/loggingproxy.py

```diff
@@ -101,6 +105,10 @@ def _handle_protocol(self):
             self.handle_pointerEvent(x, y, buttonmask)
         elif ptype == 6:
             self.handle_clientCutText(block)
+        elif ptype == 255:
+            # https://github.com/rfbproto/rfbproto/blob/master/rfbproto.rst#qemu-client-message
+            downFlag, keySym = unpack('!xHIxxxx', block)
```

Comment:
should we have a new handle_QEMU() function here rather than coding the behaviour inline? What about the audio message sub-type?

Reply:
See 6c7374c from my #232 for an alternativ implementation